### ISSUE TITLE
Make settings available from the places they affect

### ISSUE THREAD

**Issue by notramo** [qTox/qTox#5433]
Making settings available at the places they affect would be a great improvement.
## Toggle identicons
Currently requires **four steps**:
1. Go to Settings
2. Open "User Interface" tab
3. Select "Use identicons..."
4. Go back to active conversation

It could be reduced to **2 quick steps**:
1. Right click on an avatar
2. Select "Use identicons..." from the local menu

Also, it would be easier to find this setting.

## Toggle showing contact status changes
Currently requires **four steps**:
1. Go to Settings
2. Select "General"
3. Select "Show contacts' status changes"
4. Go back to active conversation

It could also be reduced to **2 quick steps**:
1. Click on a status change report
2. Select "Show contacts' ..." from right click menu

## Emoticons
Right click on an emoji in a message, or in the message input box...

## Formatting characters
Right click on a message...

## Notify
Right click in a conversation or on the tray icon...

## Toggle typing notification
Right click in the message input box...

## Reconnect
It should be also available in the tray icon menu, and in the status change button.

## Microphone, speaker volume
It should be available next to the mic/speaker mute toggle buttons in the conversation window.

**Comment by sudden6** [comment]
Personally I'm not a huge fan of distributing the settings throughout the program and prefer them in one place, but well structured.

A good example what I like: https://docs.kde.org/trunk5/en/applications/kate/ctags-global-setting.png

**Comment by notramo** [comment]
I agree that all settings should be available in one place, but **making them accessible also** from the places they affect (**not moving them**) would make qTox more user friendly and comfortable. So the current Settings window wouldn't need any change. The settings would be available in the current way, and also where they change things.

**Comment by sudden6** [comment]
Ok, then I misunderstood your proposal.
